runtime/go.mod

```diff
@@ -9,6 +9,7 @@ replace (
 
 require (
 	github.com/AdaLogics/go-fuzz-headers v0.0.0-20220903154154-e8044f6e4c72
+	github.com/darkowlzz/controller-check v0.0.0-20220902134353-b2ee0ae3566c
```

Comment:
Please replace this with the sub-package from runtime

Reply:
replaced